website/templates/emails/confirm_email_add.txt.mako

```diff
@@ -0,0 +1,9 @@
+Hello ${merge_target.fullname},
```

Comment:
> Continuation of [this comment](https://github.com/CenterForOpenScience/osf.io/pull/5000/files#r58400083)

This template appears to just be an unused copy of `website/templates/emails/confirm_merge.txt.mako`